docs/_developer_onboarding.md

```diff
@@ -276,11 +276,12 @@ Follow these steps to install and configure the plugin and templates:
 
 1. Install the AS plugin for generating modules :
    [Generate Module from Template](https://plugins.jetbrains.com/plugin/13586-generate-module-from-template)
-2. Import file templates in AS :
+2. Run the script `tools/templates/generate_templates.sh` to generate the template zip file
```

Comment:
This should be `install_templates.sh`, right?

Reply:
Yes. I have decided to rename the script actually.